source/core/options.ts

```diff
@@ -1349,13 +1349,11 @@ export default class Options {
 
 	__Note #2__: This option is not enumerable and will not be merged with the instance defaults.
 	*/
-	get json(): Record<string, any> | undefined {
+	get json(): any {
```

Comment:
@szmarczak I think it should return `unknown`.

Reply:
Well, it was `Record<string, any>` and not `Record<string, unknown>` 🤔

Reply:
The existing value was incorrect too, yes.

Reply:
Although, they should behave the same, I prefer using `unknown` whenever possible for consistency. I only use `any` when there's absolutely no other way.

Reply:
https://github.com/sindresorhus/got/commit/7ed87983477decafed6a115391a6fbafd33693fd

Reply:
:+1: